README.md

```diff
@@ -1 +1,63 @@
-Currently not working correctly. Has to be reenabled after the python 3 port of openrsx/rsb-python.
+[![Build Status](https://travis-ci.org/open-rsx/rsb-spread-python.svg?branch=master)](https://travis-ci.org/open-rsx/rsb-spread-python) [![Coverage Report](https://codecov.io/gh/open-rsx/rsb-spread-python/branch/master/graph/badge.svg)](https://codecov.io/gh/open-rsx/rsb-spread-python)
+
+# Introduction
+
+This repository contains the Python implementation of the [spread] based transport for the [Robotics Service Bus](https://github.com/open-rsx) middleware.
+
+**The full documentation for RSB can be found at <https://docs.cor-lab.de/manual/rsb-trunk/index.html>.**
```

Comment:
Link to the new documentation?

Reply:
has to be changed in all readme. I would try to do that in a batch.